server/handlers.go

```diff
@@ -1248,6 +1252,7 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", contentType)
 	w.Header().Set("Content-Length", strconv.FormatUint(contentLength, 10))
+	w.Header().Set("Vary", "Accept, X-Decrypt-Password")
```

Comment:
unless I've missed something `getHandler` is sensible only to `Range` and `X-Decrypt-Password`